base/linalg/symmetric.jl

```diff
@@ -83,8 +83,8 @@ function Hermitian(A::AbstractMatrix, uplo::Symbol=:U)
     Hermitian{eltype(A),typeof(A)}(A, char_uplo(uplo))
 end
 
-HermOrSym{T,S} = Union{Hermitian{T,S}, Symmetric{T,S}}
-RealHermSymComplexHerm{T<:Real,S} = Union{Hermitian{T,S}, Symmetric{T,S}, Hermitian{Complex{T},S}}
+const HermOrSym{T,S} = Union{Hermitian{T,S}, Symmetric{T,S}}
+const RealHermSymComplexHerm{T<:Real,S} = Union{Hermitian{T,S}, Symmetric{T,S}, Hermitian{Complex{T},S}}
```

Comment:
This can also be written
```julia
RealHermSymComplexHerm{T,S} = Union{Hermitian{T,S}, Symmetric{T,S}, Hermitian{Complex{T},S}} where S where T<:Real
```
perhaps that is more clear ? as the right hand side matches something you would find in a function signature.